docs/en/social-media.md

```diff
@@ -34,6 +34,20 @@ pageClass: routes
 
 <RouteEn author="Ovler-Young" example="/curius/links/yuu-yuu" path="/curius/links/:name" :paramsDesc="['Username, can be found in URL']"/>
 
+## Daily.dev
+
+### Popular
+
+<RouteEn author="Rjnishant530" example="/daily.dev" path="/daily.dev">
+
+### Most upvoted
+
+<RouteEn author="Rjnishant530" example="/daily.dev/upvoted" path="/daily.dev/upvoted">
+
+### Most Discussed
+
+<RouteEn author="Rjnishant530" example="/daily.dev/discussed" path="/daily.dev/discussed">
```

Comment:
Please change the namespace to `/daily` and `/daily/...`
docs: https://docs.rsshub.app/en/joinus/new-rss/before-start.html#create-a-namespace

Reply:
The website is Daily.dev, and not Daily.dev.com, so If i use the namespace daily, in future a website with daily.com , then it will be a clash that's why i thought using Daily.dev would make more sense, because i saw one more route do the same "dev.to". if you still think its bad idea, i can change it to "/daily"

Reply:
"dev.to" was written with v1 route standard which is deprecated. Those can their namespace whatever they want but that's not the case in v2 route standard.

Reply:
won't it confuse people, if i just have /daily for a website daily.dev?